### DISCUSSION TITLE
The "qutebrowser.exe has stopped working window" does not show up on crashes. What do I do to get stack traces?

### DISCUSSION THREAD

**Discussion by Unknow0059** [qutebrowser/qutebrowser#5590] [Q&A] [answered]
[I'm referring to this.](https://github.com/qutebrowser/qutebrowser/blob/master/doc/stacktrace.asciidoc#for-windows) When qutebrowser crashes - which is often - it just closes. It doesn't hang. 

Windows doesn't show me a dialog asking me whether to Wait or to Close the program.

What should I do, then?

**Comment by The-Compiler** [comment, marked as answer]
I think you should be able to open DebugDiag and add a new rule there which always dumps when `qutebrowser.exe` crashes. It's been a while since I last used it, though. I should probably try to add some documentation about that to the docs.

**Reply by The-Compiler** [reply]
@Unknow0059 I now [updated](https://github.com/qutebrowser/qutebrowser/blob/master/doc/stacktrace.asciidoc#for-windows) the stacktrace docs to include a bit more information about this.
